src/perl/lib/wtsi_clarity/epp/generic/stamper.pm

```diff
@@ -11,6 +11,7 @@ use Try::Tiny;
 
```

Comment:
Missing dependencies from the doc part:
Mojo::Collection 'c',
Try::Tiny,
'wtsi_clarity::epp',
'wtsi_clarity::util::clarity_elements',
'wtsi_clarity::util::roles::clarity_process_base'